diff -ru ../pacmanager.orig/lib/edit/PACMethod.pm ./lib/edit/PACMethod.pm
--- ../pacmanager.orig/lib/edit/PACMethod.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/edit/PACMethod.pm	2016-08-31 16:24:54.792192130 -0400
@@ -47,40 +47,40 @@
 
 no strict 'refs'; # Trick or treat!! ;)
 
-eval { require "$RealBin/lib/method/PACMethod_generic.pm"; }; die $@ if $@;
+eval { require "/usr/lib64/pacmanager/method/PACMethod_generic.pm"; }; die $@ if $@;
 $METHODS{ 'Generic Command' } = "PACMethod_generic" -> new( $CONTAINER );
 
-eval { require "$RealBin/lib/method/PACMethod_ssh.pm"; }; die $@ if $@;
+eval { require "/usr/lib64/pacmanager/method/PACMethod_ssh.pm"; }; die $@ if $@;
 $METHODS{ 'SSH' } = "PACMethod_ssh" -> new( $CONTAINER );
 
-eval { require "$RealBin/lib/method/PACMethod_mosh.pm"; }; die $@ if $@;
+eval { require "/usr/lib64/pacmanager/method/PACMethod_mosh.pm"; }; die $@ if $@;
 $METHODS{ 'MOSH' } = "PACMethod_mosh" -> new( $CONTAINER );
 
-eval { require "$RealBin/lib/method/PACMethod_cadaver.pm"; }; die $@ if $@;
+eval { require "/usr/lib64/pacmanager/method/PACMethod_cadaver.pm"; }; die $@ if $@;
 $METHODS{ 'WebDAV' } = "PACMethod_cadaver" -> new( $CONTAINER );
 
-eval { require "$RealBin/lib/method/PACMethod_sftp.pm"; }; die $@ if $@;
+eval { require "/usr/lib64/pacmanager/method/PACMethod_sftp.pm"; }; die $@ if $@;
 $METHODS{ 'SFTP' } = "PACMethod_sftp" -> new( $CONTAINER );
 
-eval { require "$RealBin/lib/method/PACMethod_ftp.pm"; }; die $@ if $@;
+eval { require "/usr/lib64/pacmanager/method/PACMethod_ftp.pm"; }; die $@ if $@;
 $METHODS{ 'FTP' } = "PACMethod_ftp" -> new( $CONTAINER );
 
-eval { require "$RealBin/lib/method/PACMethod_telnet.pm"; }; die $@ if $@;
+eval { require "/usr/lib64/pacmanager/method/PACMethod_telnet.pm"; }; die $@ if $@;
 $METHODS{ 'Telnet' } = "PACMethod_telnet" -> new( $CONTAINER );
 
-eval { require "$RealBin/lib/method/PACMethod_cu.pm"; }; die $@ if $@;
+eval { require "/usr/lib64/pacmanager/method/PACMethod_cu.pm"; }; die $@ if $@;
 $METHODS{ 'Serial (cu)' } = "PACMethod_cu" -> new( $CONTAINER );
 
-eval { require "$RealBin/lib/method/PACMethod_remote_tty.pm"; }; die $@ if $@;
+eval { require "/usr/lib64/pacmanager/method/PACMethod_remote_tty.pm"; }; die $@ if $@;
 $METHODS{ 'Serial (remote-tty)' } = "PACMethod_remote_tty" -> new( $CONTAINER );
 
-eval { require "$RealBin/lib/method/PACMethod_3270.pm"; }; die $@ if $@;
+eval { require "/usr/lib64/pacmanager/method/PACMethod_3270.pm"; }; die $@ if $@;
 $METHODS{ 'IBM 3270/5250' } = "PACMethod_3270" -> new( $CONTAINER );
 
-eval { require "$RealBin/lib/method/PACMethod_rdesktop.pm"; }; die $@ if $@;
+eval { require "/usr/lib64/pacmanager/method/PACMethod_rdesktop.pm"; }; die $@ if $@;
 $METHODS{ 'RDP (rdesktop)' } = "PACMethod_rdesktop" -> new( $CONTAINER );
 
-eval { require "$RealBin/lib/method/PACMethod_xfreerdp.pm"; }; die $@ if $@;
+eval { require "/usr/lib64/pacmanager/method/PACMethod_xfreerdp.pm"; }; die $@ if $@;
 $METHODS{ 'RDP (xfreerdp)' } = "PACMethod_xfreerdp" -> new( $CONTAINER );
 
 `which vncviewer 1>/dev/null 2>&1`;
@@ -88,10 +88,10 @@
 `vncviewer --help 2>&1 | /bin/grep TigerVNC`;
 my $tigervnc = $?;
 if ( ! $tigervnc ) {
-	eval { require "$RealBin/lib/method/PACMethod_tigervnc.pm"; }; die $@ if $@;
+	eval { require "/usr/lib64/pacmanager/method/PACMethod_tigervnc.pm"; }; die $@ if $@;
 	$METHODS{ 'VNC' } = "PACMethod_tigervnc" -> new( $CONTAINER );
 } elsif ( ! $xtightvncviewer ) {
-	eval { require "$RealBin/lib/method/PACMethod_vncviewer.pm"; }; die $@ if $@;
+	eval { require "/usr/lib64/pacmanager/method/PACMethod_vncviewer.pm"; }; die $@ if $@;
 	$METHODS{ 'VNC' } = "PACMethod_vncviewer" -> new( $CONTAINER );
 }
 
diff -ru ../pacmanager.orig/lib/method/PACMethod_3270.pm ./lib/method/PACMethod_3270.pm
--- ../pacmanager.orig/lib/method/PACMethod_3270.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/method/PACMethod_3270.pm	2016-08-31 16:24:54.776191929 -0400
@@ -38,7 +38,7 @@
 ###################################################################
 # Define GLOBAL CLASS variables
 
-my $RES_DIR = $RealBin . '/res';
+my $RES_DIR = '/usr/share/pacmanager';
 
 # END: Define GLOBAL CLASS variables
 ###################################################################
diff -ru ../pacmanager.orig/lib/method/PACMethod_cadaver.pm ./lib/method/PACMethod_cadaver.pm
--- ../pacmanager.orig/lib/method/PACMethod_cadaver.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/method/PACMethod_cadaver.pm	2016-08-31 16:24:54.774191904 -0400
@@ -38,7 +38,7 @@
 ###################################################################
 # Define GLOBAL CLASS variables
 
-my $RES_DIR = $RealBin . '/res';
+my $RES_DIR = '/usr/share/pacmanager';
 
 # END: Define GLOBAL CLASS variables
 ###################################################################
diff -ru ../pacmanager.orig/lib/method/PACMethod_cu.pm ./lib/method/PACMethod_cu.pm
--- ../pacmanager.orig/lib/method/PACMethod_cu.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/method/PACMethod_cu.pm	2016-08-31 16:24:54.772191879 -0400
@@ -40,7 +40,7 @@
 ###################################################################
 # Define GLOBAL CLASS variables
 
-my $RES_DIR = $RealBin . '/res';
+my $RES_DIR = '/usr/share/pacmanager';
 my %PARITY = ( 'even' => 0, 'none' => 1, 'odd' => 2 );
 
 # END: Define GLOBAL CLASS variables
diff -ru ../pacmanager.orig/lib/method/PACMethod_ftp.pm ./lib/method/PACMethod_ftp.pm
--- ../pacmanager.orig/lib/method/PACMethod_ftp.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/method/PACMethod_ftp.pm	2016-08-31 16:24:54.768191829 -0400
@@ -38,7 +38,7 @@
 ###################################################################
 # Define GLOBAL CLASS variables
 
-my $RES_DIR = $RealBin . '/res';
+my $RES_DIR = '/usr/share/pacmanager';
 
 # END: Define GLOBAL CLASS variables
 ###################################################################
diff -ru ../pacmanager.orig/lib/method/PACMethod_generic.pm ./lib/method/PACMethod_generic.pm
--- ../pacmanager.orig/lib/method/PACMethod_generic.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/method/PACMethod_generic.pm	2016-08-31 16:24:54.776191929 -0400
@@ -38,7 +38,7 @@
 ###################################################################
 # Define GLOBAL CLASS variables
 
-my $RES_DIR = $RealBin . '/res';
+my $RES_DIR = '/usr/share/pacmanager';
 
 # END: Define GLOBAL CLASS variables
 ###################################################################
diff -ru ../pacmanager.orig/lib/method/PACMethod_mosh.pm ./lib/method/PACMethod_mosh.pm
--- ../pacmanager.orig/lib/method/PACMethod_mosh.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/method/PACMethod_mosh.pm	2016-08-31 16:24:54.770191854 -0400
@@ -38,7 +38,7 @@
 ###################################################################
 # Define GLOBAL CLASS variables
 
-my $RES_DIR = $RealBin . '/res';
+my $RES_DIR = '/usr/share/pacmanager';
 
 # END: Define GLOBAL CLASS variables
 ###################################################################
diff -ru ../pacmanager.orig/lib/method/PACMethod_rdesktop.pm ./lib/method/PACMethod_rdesktop.pm
--- ../pacmanager.orig/lib/method/PACMethod_rdesktop.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/method/PACMethod_rdesktop.pm	2016-08-31 16:24:54.775191917 -0400
@@ -42,7 +42,7 @@
 my %BPP 		= ( 8 => 0, 15 => 1, 16 => 2, 24 => 3 );
 my %SOUND		= ( 'local' => 0, 'off' => 1, 'remote' => 2 );
 
-my $RES_DIR = $RealBin . '/res';
+my $RES_DIR = '/usr/share/pacmanager';
 
 # END: Define GLOBAL CLASS variables
 ###################################################################
diff -ru ../pacmanager.orig/lib/method/PACMethod_remote_tty.pm ./lib/method/PACMethod_remote_tty.pm
--- ../pacmanager.orig/lib/method/PACMethod_remote_tty.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/method/PACMethod_remote_tty.pm	2016-08-31 16:24:54.768191829 -0400
@@ -40,7 +40,7 @@
 ###################################################################
 # Define GLOBAL CLASS variables
 
-my $RES_DIR = $RealBin . '/res';
+my $RES_DIR = '/usr/share/pacmanager';
 
 # END: Define GLOBAL CLASS variables
 ###################################################################
diff -ru ../pacmanager.orig/lib/method/PACMethod_sftp.pm ./lib/method/PACMethod_sftp.pm
--- ../pacmanager.orig/lib/method/PACMethod_sftp.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/method/PACMethod_sftp.pm	2016-08-31 16:24:54.773191891 -0400
@@ -42,7 +42,7 @@
 # Define GLOBAL CLASS variables
 
 my %SSH_VERSION = ( 1 => 0, 'any' => 1 );
-my $RES_DIR = $RealBin . '/res';
+my $RES_DIR = '/usr/share/pacmanager';
 my %SSH_ADV_OPTS = (
 	'A' => [ 'AddressFamily' ],
 	'B' => [ 'BatchMode', 'BindAddress' ],
diff -ru ../pacmanager.orig/lib/method/PACMethod_ssh.pm ./lib/method/PACMethod_ssh.pm
--- ../pacmanager.orig/lib/method/PACMethod_ssh.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/method/PACMethod_ssh.pm	2016-08-31 16:24:54.772191879 -0400
@@ -65,7 +65,7 @@
 	'V' => [ 'VerifyHostKeyDNS', 'VisualHostKey' ],
 	'X' => [ 'XAuthLocation' ]
 );
-my $RES_DIR = $RealBin . '/res';
+my $RES_DIR = '/usr/share/pacmanager';
 
 # END: Define GLOBAL CLASS variables
 ###################################################################
diff -ru ../pacmanager.orig/lib/method/PACMethod_telnet.pm ./lib/method/PACMethod_telnet.pm
--- ../pacmanager.orig/lib/method/PACMethod_telnet.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/method/PACMethod_telnet.pm	2016-08-31 16:24:54.769191841 -0400
@@ -41,7 +41,7 @@
 # Define GLOBAL CLASS variables
 
 my %IP_PROTOCOL = ( 4 => 0, 6 => 1, 'any' => 2 );
-my $RES_DIR = $RealBin . '/res';
+my $RES_DIR = '/usr/share/pacmanager';
 
 # END: Define GLOBAL CLASS variables
 ###################################################################
diff -ru ../pacmanager.orig/lib/method/PACMethod_tigervnc.pm ./lib/method/PACMethod_tigervnc.pm
--- ../pacmanager.orig/lib/method/PACMethod_tigervnc.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/method/PACMethod_tigervnc.pm	2016-08-31 16:24:54.775191917 -0400
@@ -38,7 +38,7 @@
 ###################################################################
 # Define GLOBAL CLASS variables
 
-my $RES_DIR = $RealBin . '/res';
+my $RES_DIR = '/usr/share/pacmanager';
 my %COLOURS = ( 8 => 0, 64 => 1, 256 => 2, 'AutoSelect' => 3 );
 my %REVCOLOURS = ( 0 => 0, 1 => 1, 2 => 2, 'AutoSelect' => 3 );
 
diff -ru ../pacmanager.orig/lib/method/PACMethod_vncviewer.pm ./lib/method/PACMethod_vncviewer.pm
--- ../pacmanager.orig/lib/method/PACMethod_vncviewer.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/method/PACMethod_vncviewer.pm	2016-08-31 16:24:54.769191841 -0400
@@ -40,7 +40,7 @@
 
 my $ENCODINGS = "Tight Zlib Hextile CoRRE RRE CopyRect Raw";
 my %DEPTH = ( 8 => 0, 15 => 1, 16 => 2, 24 => 3, 32 => 4, 'default' => 5 );
-my $RES_DIR = $RealBin . '/res';
+my $RES_DIR = '/usr/share/pacmanager';
 
 # END: Define GLOBAL CLASS variables
 ###################################################################
diff -ru ../pacmanager.orig/lib/method/PACMethod_xfreerdp.pm ./lib/method/PACMethod_xfreerdp.pm
--- ../pacmanager.orig/lib/method/PACMethod_xfreerdp.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/method/PACMethod_xfreerdp.pm	2016-08-31 16:24:54.768191829 -0400
@@ -40,7 +40,7 @@
 
 my %BPP 		= ( 8 => 0, 15 => 1, 16 => 2, 24 => 3 );
 
-my $RES_DIR = $RealBin . '/res';
+my $RES_DIR = '/usr/share/pacmanager';
 
 # END: Define GLOBAL CLASS variables
 ###################################################################
diff -ru ../pacmanager.orig/lib/PACCluster.pm ./lib/PACCluster.pm
--- ../pacmanager.orig/lib/PACCluster.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/PACCluster.pm	2016-08-31 16:24:54.706191051 -0400
@@ -25,7 +25,7 @@
 # Import Modules
 
 use FindBin qw ( $RealBin $Bin $Script );
-use lib $RealBin . '/lib', $RealBin . '/lib/ex';
+use lib '/usr/lib64/pacmanager', '/usr/lib64/pacmanager/ex', '/usr/lib64/pacmanager/edit';
 
 # Standard
 use strict;
@@ -52,12 +52,12 @@
 
 my $CFG_DIR			= $ENV{'HOME'} . '/.config/pac';
 my $CLUSTERS_FILE	= $CFG_DIR . '/pac_clusters.nfreeze';
-my $GROUPICONCLOSED	= _pixBufFromFile( $RealBin . '/res/pac_group_closed_16x16.png' );
-my $GROUPICON_ROOT	= _pixBufFromFile( $RealBin . '/res/pac_group.png' );
-my $AUTOCLUSTERICON	= _pixBufFromFile( $RealBin . '/res/pac_cluster_auto.png' );
-my $ICON_ON			= Gtk2::Gdk::Pixbuf -> new_from_file_at_scale( $RealBin . '/res/pac_terminal16x16.png', 16, 16, 0 );
-my $ICON_OFF		= Gtk2::Gdk::Pixbuf -> new_from_file_at_scale( $RealBin . '/res/pac_terminal_x16x16.png', 16, 16, 0 );
-my $BANNER			= Gtk2::Image -> new_from_file( $RealBin . '/res/pac_banner_cluster.png' );
+my $GROUPICONCLOSED	= _pixBufFromFile( '/usr/share/pacmanager/pac_group_closed_16x16.png' );
+my $GROUPICON_ROOT	= _pixBufFromFile( '/usr/share/pacmanager/pac_group.png' );
+my $AUTOCLUSTERICON	= _pixBufFromFile( '/usr/share/pacmanager/pac_cluster_auto.png' );
+my $ICON_ON			= Gtk2::Gdk::Pixbuf -> new_from_file_at_scale( '/usr/share/pacmanager/pac_terminal16x16.png', 16, 16, 0 );
+my $ICON_OFF		= Gtk2::Gdk::Pixbuf -> new_from_file_at_scale( '/usr/share/pacmanager/pac_terminal_x16x16.png', 16, 16, 0 );
+my $BANNER			= Gtk2::Image -> new_from_file( '/usr/share/pacmanager/pac_banner_cluster.png' );
 # END: Define GLOBAL CLASS variables
 ###################################################################
 
diff -ru ../pacmanager.orig/lib/PACConfig.pm ./lib/PACConfig.pm
--- ../pacmanager.orig/lib/PACConfig.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/PACConfig.pm	2016-08-31 16:24:54.788192080 -0400
@@ -54,11 +54,11 @@
 
 my $APPNAME			= $PACUtils::APPNAME;
 my $APPVERSION		= $PACUtils::APPVERSION;
-my $AUTOSTART_FILE	= $RealBin . '/res/pac_start.desktop';
+my $AUTOSTART_FILE	= '/usr/share/pacmanager/pac_start.desktop';
 
-my $GLADE_FILE		= $RealBin . '/res/pac.glade';
+my $GLADE_FILE		= '/usr/share/pacmanager/pac.glade';
 my $CFG_DIR			= $ENV{'HOME'} . '/.config/pac';
-my $RES_DIR			= $RealBin . '/res';
+my $RES_DIR			= '/usr/share/pacmanager';
 
 # Connect to Gnome's GConf
 my $GCONF			= Gnome2::GConf::Client -> get_default;
diff -ru ../pacmanager.orig/lib/pac_conn ./lib/pac_conn
--- ../pacmanager.orig/lib/pac_conn	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/pac_conn	2016-08-31 16:24:54.713191139 -0400
@@ -28,7 +28,7 @@
 use warnings;
 
 use FindBin qw ( $RealBin $Bin $Script );
-use lib $RealBin, $RealBin . '/ex', $RealBin . '/edit';
+use lib '/usr/lib64/pacmanager', '/usr/lib64/pacmanager/ex', '/usr/lib64/pacmanager/edit';
 use Storable qw ( dclone thaw retrieve fd_retrieve nstore_fd );
 use Expect;
 use Gnome2::GConf;
diff -ru ../pacmanager.orig/lib/PACEdit.pm ./lib/PACEdit.pm
--- ../pacmanager.orig/lib/PACEdit.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/PACEdit.pm	2016-08-31 16:24:54.719191214 -0400
@@ -25,7 +25,7 @@
 # Import Modules
 
 use FindBin qw ( $RealBin $Bin $Script );
-use lib $RealBin . '/lib/edit';
+use lib '/usr/lib64/pacmanager', '/usr/lib64/pacmanager/ex', '/usr/lib64/pacmanager/edit';
 
 # Standard
 use strict;
@@ -58,7 +58,7 @@
 
 my $APPNAME			= $PACUtils::APPNAME;
 my $APPVERSION		= $PACUtils::APPVERSION;
-my $RES_DIR			= $RealBin . '/res';
+my $RES_DIR			= '/usr/share/pacmanager';
 my $AUTOSTART_FILE	= $RES_DIR . '/pac_start.desktop';
 
 my $GLADE_FILE		= $RES_DIR . '/pac.glade';
diff -ru ../pacmanager.orig/lib/PACMain.pm ./lib/PACMain.pm
--- ../pacmanager.orig/lib/PACMain.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/PACMain.pm	2016-08-31 16:24:54.785192042 -0400
@@ -26,7 +26,7 @@
 
 use FindBin qw ( $RealBin $Bin $Script );
 my $REALBIN = $RealBin;
-use lib $RealBin . '/lib', $RealBin . '/lib/ex';
+use lib '/usr/lib64/pacmanager', '/usr/lib64/pacmanager/ex', '/usr/lib64/pacmanager/edit';
 
 # Standard
 use strict;
@@ -72,13 +72,13 @@
 
 my $APPNAME			= $PACUtils::APPNAME;
 my $APPVERSION		= $PACUtils::APPVERSION;
-my $AUTOSTART_FILE	= $RealBin . '/res/pac_start.desktop';
-my $RES_DIR			= $RealBin . '/res';
+my $AUTOSTART_FILE	= '/usr/share/pacmanager/pac_start.desktop';
+my $RES_DIR			= '/usr/share/pacmanager';
 
 # Register PAC's icons on Gtk
 &_registerPACIcons;
 
-my $INIT_CFG_FILE		= $RealBin . '/res/pac.yml';
+my $INIT_CFG_FILE		= '/usr/share/pacmanager/pac.yml';
 my $CFG_DIR				= $ENV{'HOME'} . '/.config/pac';
 my $CFG_FILE			= $CFG_DIR . '/pac.yml';
 our $R_CFG_FILE			= '';
@@ -90,12 +90,12 @@
 my $PAC_START_TOTAL		= 6;
 
 my $APPICON			= $RES_DIR . '/pac64x64.png';
-my $AUTOCLUSTERICON	= _pixBufFromFile( $RealBin . '/res/pac_cluster_auto.png' );
-my $CLUSTERICON		= _pixBufFromFile( $RealBin . '/res/pac_cluster_manager.png' );
-my $GROUPICON_ROOT	= _pixBufFromFile( $RealBin . '/res/pac_group.png' );
-my $GROUPICON		= _pixBufFromFile( $RealBin . '/res/pac_group_open_16x16.png' );
-my $GROUPICONOPEN	= _pixBufFromFile( $RealBin . '/res/pac_group_open_16x16.png' );
-my $GROUPICONCLOSED	= _pixBufFromFile( $RealBin . '/res/pac_group_closed_16x16.png' );
+my $AUTOCLUSTERICON	= _pixBufFromFile( '/usr/share/pacmanager/pac_cluster_auto.png' );
+my $CLUSTERICON		= _pixBufFromFile( '/usr/share/pacmanager/pac_cluster_manager.png' );
+my $GROUPICON_ROOT	= _pixBufFromFile( '/usr/share/pacmanager/pac_group.png' );
+my $GROUPICON		= _pixBufFromFile( '/usr/share/pacmanager/pac_group_open_16x16.png' );
+my $GROUPICONOPEN	= _pixBufFromFile( '/usr/share/pacmanager/pac_group_open_16x16.png' );
+my $GROUPICONCLOSED	= _pixBufFromFile( '/usr/share/pacmanager/pac_group_closed_16x16.png' );
 
 my $CHECK_VERSION	= 0;
 my $NEW_VERSION		= 0;
diff -ru ../pacmanager.orig/lib/PACPCC.pm ./lib/PACPCC.pm
--- ../pacmanager.orig/lib/PACPCC.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/PACPCC.pm	2016-08-31 16:24:54.718191202 -0400
@@ -46,7 +46,7 @@
 
 my $APPNAME			= $PACUtils::APPNAME;
 my $APPVERSION		= $PACUtils::APPVERSION;
-my $APPICON			= $RealBin . '/res/pac64x64.png';
+my $APPICON			= '/usr/share/pacmanager/pac64x64.png';
 my $CFG_DIR			= $ENV{'HOME'} . '/.config/pac';
 
 my %LANG;
diff -ru ../pacmanager.orig/lib/PACPipe.pm ./lib/PACPipe.pm
--- ../pacmanager.orig/lib/PACPipe.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/PACPipe.pm	2016-08-31 16:24:54.720191227 -0400
@@ -44,7 +44,7 @@
 
 my $APPNAME			= $PACUtils::APPNAME;
 my $APPVERSION		= $PACUtils::APPVERSION;
-my $APPICON			= $RealBin . '/res/pac64x64.png';
+my $APPICON			= '/usr/share/pacmanager/pac64x64.png';
 
 # END: Define GLOBAL CLASS variables
 ###################################################################
diff -ru ../pacmanager.orig/lib/PACScreenshots.pm ./lib/PACScreenshots.pm
--- ../pacmanager.orig/lib/PACScreenshots.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/PACScreenshots.pm	2016-08-31 16:24:54.715191164 -0400
@@ -45,7 +45,7 @@
 
 my $APPNAME		= $PACUtils::APPNAME;
 my $APPVERSION	= $PACUtils::APPVERSION;
-my $APPICON		= $RealBin . '/res/pac64x64.png';
+my $APPICON		= '/usr/share/pacmanager/pac64x64.png';
 my $CFG_DIR		= $ENV{'HOME'} . '/.config/pac';
 
 # END: Define GLOBAL CLASS variables
diff -ru ../pacmanager.orig/lib/PACScripts.pm ./lib/PACScripts.pm
--- ../pacmanager.orig/lib/PACScripts.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/PACScripts.pm	2016-08-31 16:24:54.766191804 -0400
@@ -52,8 +52,8 @@
 
 my $APPNAME			= $PACUtils::APPNAME;
 my $APPVERSION		= $PACUtils::APPVERSION;
-my $APPICON			= $RealBin . '/res/pac64x64.png';
-my $BANNER			= $RealBin . '/res/pac_banner_scripts.png';
+my $APPICON			= '/usr/share/pacmanager/pac64x64.png';
+my $BANNER			= '/usr/share/pacmanager/pac_banner_scripts.png';
 my $CFG_DIR			= $ENV{'HOME'} . '/.config/pac';
 my $SCRIPTS_DIR		= $CFG_DIR . '/scripts';
 
diff -ru ../pacmanager.orig/lib/PACStatistics.pm ./lib/PACStatistics.pm
--- ../pacmanager.orig/lib/PACStatistics.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/PACStatistics.pm	2016-08-31 16:24:54.786192054 -0400
@@ -47,7 +47,7 @@
 
 my $APPNAME		= $PACUtils::APPNAME;
 my $APPVERSION	= $PACUtils::APPVERSION;
-my $APPICON		= $RealBin . '/res/pac64x64.png';
+my $APPICON		= '/usr/share/pacmanager/pac64x64.png';
 my $CFG_DIR		= $ENV{'HOME'} . '/.config/pac';
 my $STATS_FILE	= $CFG_DIR . '/pac_stats.nfreeze';
 
diff -ru ../pacmanager.orig/lib/PACTerminal.pm ./lib/PACTerminal.pm
--- ../pacmanager.orig/lib/PACTerminal.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/PACTerminal.pm	2016-08-31 16:24:54.737191440 -0400
@@ -29,7 +29,7 @@
 use warnings;
 
 use FindBin qw ( $RealBin $Bin $Script );
-use lib $RealBin . '/lib', $RealBin . '/lib/ex';
+use lib '/usr/lib64/pacmanager', '/usr/lib64/pacmanager/ex', '/usr/lib64/pacmanager/edit';
 use Storable qw ( dclone nstore nstore_fd fd_retrieve );
 use POSIX qw ( strftime );
 use File::Copy;
@@ -55,11 +55,11 @@
 
 my $APPNAME		= $PACUtils::APPNAME;
 my $APPVERSION	= $PACUtils::APPVERSION;
-my $APPICON		= $RealBin . '/res/pac64x64.png';
+my $APPICON		= '/usr/share/pacmanager/pac64x64.png';
 my $CFG_DIR		= $ENV{'HOME'} . '/.config/pac';
 
 my $PERL_BIN	= '/usr/bin/perl';
-my $PAC_CONN	= $RealBin . '/lib/pac_conn';
+my $PAC_CONN	= '/usr/lib64/pacmanager/pac_conn';
 
 my $SHELL_BIN	= -x '/bin/sh' ? '/bin/sh' : '/bin/bash';
 my $SHELL_NAME	= -x '/bin/sh' ? 'sh' : 'bash';
@@ -1902,7 +1902,7 @@
 			);
 		}
 	} else {
-		defined $$self{_WINDOWTERMINAL} and $$self{_WINDOWTERMINAL} -> set_icon_from_file( $$self{CONNECTED} ? $RealBin . '/res/pac_terminal64x64.png' : $RealBin . '/res/pac_terminal_x64x64.png' );
+		defined $$self{_WINDOWTERMINAL} and $$self{_WINDOWTERMINAL} -> set_icon_from_file( $$self{CONNECTED} ? '/usr/share/pacmanager/pac_terminal64x64.png' : '/usr/share/pacmanager/pac_terminal_x64x64.png' );
 	}
 	
 	# Once checked the availability of new data, reset its value
diff -ru ../pacmanager.orig/lib/PACTray.pm ./lib/PACTray.pm
--- ../pacmanager.orig/lib/PACTray.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/PACTray.pm	2016-08-31 16:24:54.795192167 -0400
@@ -43,9 +43,9 @@
 
 my $APPNAME			= $PACUtils::APPNAME;
 my $APPVERSION		= $PACUtils::APPVERSION;
-my $APPICON			= $RealBin . '/res/pac64x64.png';
-my $TRAYICON		= $RealBin . '/res/pac_tray.png';
-my $GROUPICON_ROOT	= _pixBufFromFile( $RealBin . '/res/pac_group.png' );
+my $APPICON			= '/usr/share/pacmanager/pac64x64.png';
+my $TRAYICON		= '/usr/share/pacmanager/pac_tray.png';
+my $GROUPICON_ROOT	= _pixBufFromFile( '/usr/share/pacmanager/pac_group.png' );
 # END: Define GLOBAL CLASS variables
 ###################################################################
 
@@ -61,7 +61,7 @@
 	
 	$self -> {_TRAY}	= undef;
 	
-	$TRAYICON = $RealBin . '/res/pac_tray' . ( $$self{_MAIN}{_CFG}{defaults}{'use bw icon'} ? '_bw.png' : '.png' );
+	$TRAYICON = '/usr/share/pacmanager/pac_tray' . ( $$self{_MAIN}{_CFG}{defaults}{'use bw icon'} ? '_bw.png' : '.png' );
 
 	# Build the GUI
 	_initGUI( $self ) or return 0;
diff -ru ../pacmanager.orig/lib/PACTrayUnity.pm ./lib/PACTrayUnity.pm
--- ../pacmanager.orig/lib/PACTrayUnity.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/PACTrayUnity.pm	2016-08-31 16:24:54.720191227 -0400
@@ -44,9 +44,9 @@
 
 my $APPNAME			= $PACUtils::APPNAME;
 my $APPVERSION		= $PACUtils::APPVERSION;
-my $APPICON			= $RealBin . '/res/pac64x64.png';
-my $TRAYICON		= $RealBin . '/res/pac_tray.png';
-my $GROUPICON_ROOT	= _pixBufFromFile( $RealBin . '/res/pac_group.png' );
+my $APPICON			= '/usr/share/pacmanager/pac64x64.png';
+my $TRAYICON		= '/usr/share/pacmanager/pac_tray.png';
+my $GROUPICON_ROOT	= _pixBufFromFile( '/usr/share/pacmanager/pac_group.png' );
 # END: Define GLOBAL CLASS variables
 ###################################################################
 
@@ -88,7 +88,7 @@
 	my $self = shift;
 	
 	$$self{_TRAY} = Gtk2::AppIndicator -> new( 'pac', $TRAYICON );
-	$$self{_TRAY} -> set_icon_theme_path( $RealBin . '/res' );
+	$$self{_TRAY} -> set_icon_theme_path( '/usr/share/pacmanager' );
 	$$self{_TRAY} -> set_active;
 	$$self{_MAIN}{_CFG}{'tmp'}{'tray available'} = ! $@;
 	return 1;
diff -ru ../pacmanager.orig/lib/PACUtils.pm ./lib/PACUtils.pm
--- ../pacmanager.orig/lib/PACUtils.pm	2015-11-04 16:46:13.000000000 -0500
+++ ./lib/PACUtils.pm	2016-08-31 16:24:54.728191327 -0400
@@ -114,7 +114,7 @@
 elsif	( $ARCH_TMP =~ /armv7l/gio )	{ $ARCH = 'ARMV7L'; }
 elsif	( $ARCH_TMP =~ /arm/gio )		{ $ARCH = 'ARM'; }
 else									{ $ARCH = 32; }
-my $RES_DIR			= $RealBin . '/res';
+my $RES_DIR			= '/usr/share/pacmanager';
 my $SPLASH_IMG		= $RES_DIR . '/pac256x256.jpg';
 my $CFG_DIR			= $ENV{'HOME'} . '/.config/pac';
 my $CFG_FILE		= $CFG_DIR . '/pac.yml';
@@ -185,7 +185,7 @@
 
 # Gnome2 - VTE (here be dragons...)
 eval q {
-	unshift( @INC, $RealBin . '/lib/ex/vte' . $PACUtils::ARCH );
+	unshift( @INC, '/usr/lib64/pacmanager/ex/vte' . $PACUtils::ARCH );
 	
 	package Gnome2::Vte;
 	
diff -ru ../pacmanager.orig/pac ./pac
--- ../pacmanager.orig/pac	2015-11-04 16:46:13.000000000 -0500
+++ ./pac	2016-08-31 16:24:54.850192857 -0400
@@ -52,7 +52,7 @@
 # START: Import Modules
 
 use FindBin qw ( $RealBin $Bin $Script );
-use lib $RealBin . '/lib', $RealBin . '/lib/ex', $RealBin . '/lib/edit';
+use lib '/usr/lib64/pacmanager', '/usr/lib64/pacmanager/ex', '/usr/lib64/pacmanager/edit';
 use File::Copy;
 
 # Standard
@@ -103,10 +103,10 @@
 -d $CFG_DIR . '/sockets'				or mkdir( $CFG_DIR . '/sockets' );
 -d $CFG_DIR . '/tmp'					or mkdir( $CFG_DIR . '/tmp' );
 -d $CFG_DIR . '/scripts'				or mkdir( $CFG_DIR . '/scripts' );
--d $CFG_DIR . '/scripts/sample1.pl'		or copy( $RealBin . '/res/sample1.pl', $CFG_DIR . '/scripts/' );
--d $CFG_DIR . '/scripts/sample2.pl'		or copy( $RealBin . '/res/sample2.pl', $CFG_DIR . '/scripts/' );
--d $CFG_DIR . '/scripts/sample3.pl'		or copy( $RealBin . '/res/sample3.pl', $CFG_DIR . '/scripts/' );
--d $CFG_DIR . '/scripts/sample4.pl'		or copy( $RealBin . '/res/sample4.pl', $CFG_DIR . '/scripts/' );
+-d $CFG_DIR . '/scripts/sample1.pl'		or copy( '/usr/share/pacmanager/sample1.pl', $CFG_DIR . '/scripts/' );
+-d $CFG_DIR . '/scripts/sample2.pl'		or copy( '/usr/share/pacmanager/sample2.pl', $CFG_DIR . '/scripts/' );
+-d $CFG_DIR . '/scripts/sample3.pl'		or copy( '/usr/share/pacmanager/sample3.pl', $CFG_DIR . '/scripts/' );
+-d $CFG_DIR . '/scripts/sample4.pl'		or copy( '/usr/share/pacmanager/sample4.pl', $CFG_DIR . '/scripts/' );
 
 # Start PAC!!
 if ( ! ( $PAC = PACMain -> new( @ARGV ) ) ) {
